students/jeewon-shin/westagram/user/views.py

```diff
@@ -0,0 +1,26 @@
+import json
+from django.http  import JsonResponse
+
+from django.views import View
+from user.models  import User
+
+class SignupView(View):
+    def post(self, request):
+        length_password = 8
```

Comment:
상수는 보통 대문자로만 이루어져 있습니다.
```suggestion
        length_password = 8
```
```suggestion
        PASSWORD_LENGTH = 8
```